packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -210,7 +210,7 @@ export default defineComponent({
     },
     quotaUsagePercent() {
       return this.useLegacyQuota
-        ? this.quota.relative
+        ? parseFloat(this.quota.relative.toFixed(2))
```

Comment:
Only parse this.quota.relative to float and then to a toFixed(2) on the result.

Reply:
I did that initially but then `quotaUsagePercent` is `string | number` and it gets messy further down where we're calculating how much free quota is left to change the color of the OcProgress component. 
The alternative was `parseFloat(parseFloat(this.quota.relative).toFixed(2))` but that also didn't really feel right...

Reply:
We'll go ahead as-is and can adapt (= add another `parseFloat`) if EOS/Reva requires it